library/general/src/lib/installation/proposal_client.rb

```diff
@@ -132,6 +137,11 @@ def run
     #     The caller will format each item as a HTML list item (`<li>`). The
     #     proposal can contain hyperlinks with IDs listed in the list `links`.
     #
+    #   * **`"label_proposal"`** [Array<String>, nil]
```

Comment:
why introduce another key in result? I suggest to reuse "raw_proposal" which already have right format and when "cramped_label_mode" is used, then it is hard requirement to return it. What do you think? I think we should add only unnecessary complexity.
btw cramped_label_mode is not easily remembered key. what about "minimal" or "shrinked_mode" ?